File: OOP/Hare_Krishna/src/Hare_Krishna/Prac_11.java
package Hare_Krishna;

import java.util.Scanner;

public class Prac_11
{
    private static void dplcte()
    {
        Scanner scnr = new Scanner(System.in);
        System.out.print("\nEnter the size of String array : ");
        int n = scnr.nextInt();
        String[] in = new String[n];
        System.out.print("\nEnter the Strings : ");
        for (int i=0; i<n; i++)
            in[i] = scnr.next();

        for (int a=0; a<n; a++)
        {
            for (int b=a+1; b<n; b++)
            {
                if (in[a].equals(in[b]))
                    System.out.println(in[b]);
            }
        }
    }
    public static void main(String... s)
    {
        dplcte();
    }
}

File: DS/Haribol/src/prac_11.kt
package Hare_Krishna

import java.util.Scanner

class prac_11(var size : Int, var front : Int = -1, var rear : Int = -1)
{
    var qu = IntArray(size)
    fun enq(y: Int)
    {
        if (front == -1 && rear == -1)
        {
            front=0
            rear=0
            qu[rear]=y
        }
        else if((rear + 1)%qu.size == front)
        {
            println("!!Full!!")
        }
        else
        {
            qu[rear] = y
            rear = (rear+1) % qu.size
        }
    }
    fun deq()
    {
        if (front == -1 && rear == -1)
        {
            println("!!Empty!!")
        }
        else if (front == rear)
        {
            front = -1
            rear = -1
        }
        else
        {
            front = (front+1) % qu.size
        }
    }
    fun dsply()
    {
        if (front == -1 && rear == -1)
        {
            println("!!Empty!!")
        }
        while(front<=rear)
        {
            println("->" + qu[i].toString())
            front = (front+1) % qu.size
        }
    }
}

fun main()
{
    val q = prac_10(size = 5)
    val ysl = Scanner(System.`in`)

    q.enq(ysl.nextInt())
    q.enq(ysl.nextInt())
    q.enq(ysl.nextInt())

    q.deq()
    q.deq()

    q.enq(ysl.nextInt())

    q.deq()

    q.enq(ysl.nextInt())
    q.dsply()

    q.enq(ysl.nextInt())

}